mod_form.php

```diff
@@ -264,4 +273,16 @@ public function data_preprocessing(&$defaultvalues) {
             ];
         }
     }
+
+    public function validation($data, $files) {
```

Comment:
The following error is a little confusing as setting up to Single hides "Multiple signup method". Should w just aromatically change it or ignore if it's set to singe? 

![image](https://user-images.githubusercontent.com/4626431/195738374-546bd496-9330-4622-a0f7-ad72ddaedb4c.png)

Reply:
I actually needed to save activity twice to go passed this error

Reply:
Good point, I've added a fixup commit that removes this validation and simply forces the activity to be in a valid state when saved.

Reply:
Thank you! works better now